apps/src/lab2/views/components/predict-summary.module.scss

```diff
@@ -0,0 +1,35 @@
+@import 'color.scss';
+
+.predictSummaryContainer {
+  margin-bottom: 15px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 10px;
+}
+
+.responses {
+  display: flex;
+}
+
+.responseIcon {
+  padding-right: 4px;
+
+  :global(.fa) {
```

Comment:
for my understanding - what does this do?

Reply:
this applies styling to the `fa` class specifically, rather than turning it into a css module class (which adds extra stuff to the class name so it is unique). We need this to style the font awesome icon. Since it's inside a css module-ized class it should be safe to do. We are doing something similar in the [old version](https://github.com/code-dot-org/code-dot-org/blob/0bf7d37763005917509eeac86abc6d788a09170a/apps/src/templates/levelSummary/summary-entry-point.module.scss#L40) of this (which is how I learned to do it)